src/main/scala/cromwell/Main.scala

```diff
@@ -257,22 +257,10 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
 
   def usageAndExit(): Int = {
     println(
-      """
+      s"""
```

Comment:
Based on discussion with @scottfrazer we had those WDL-specific endpoints just hardwire local backend instead of changing all the related documentation considering they're departing this jar in a couple of weeks. I'll remove the s